apps/spruce/src/components/Spawn/spawnHostModal/getFormSchema.tsx

```diff
@@ -365,6 +370,22 @@ export const getFormSchema = ({
           "ui:disabled": useProjectSetupScript,
           "ui:data-cy": "setup-script-checkbox",
         },
+        warning: {
+          "ui:showLabel": false,
+          "ui:warnings": [
+            <>
+              This script is not guaranteed to run or succeed upon host startup.
+              Consider opting into{" "}
+              <StyledRouterLink
+                to={getPreferencesRoute(PreferencesTabRoutes.Notifications)}
+              >
+                “Spawn host outcome” notifications
+              </StyledRouterLink>{" "}
+              to monitor the state of setup scripts. If further investigation is
+              required, details can be found in the host’s logs.
```

Comment:
super nit: this apostrophe seems to be a special apostrophe, not sure if that was intentional or not
```suggestion
              required, details can be found in the host&apos;s logs.
```